pypesto/optimize/optimizer.py

```diff
@@ -555,3 +560,51 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        sigma0 = 0.25 * np.median(ub)
+
+        # take the smallest lower bound and the biggest upper bound as default parameter bounds
+        self.options['bounds'] = [sorted(lb)[0], sorted(ub, reverse=True)[0]]
```

Comment:
Hm, this is still wrong. You do want the full lower and upper bounds, for the n-dimensional vector. Isn't that possible?

Reply:
I fixed it

Reply:
why not `np.max` and `np.min`?

Reply:
looks cleaner to me and should be faster...